RecoTracker/LSTCore/src/alpaka/Kernels.h

```diff
@@ -162,7 +162,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
             float eta2 = __H2F(quintuplets.eta()[jx]);
             float phi2 = __H2F(quintuplets.phi()[jx]);
             float dEta = alpaka::math::abs(acc, eta1 - eta2);
-            float dPhi = calculate_dPhi(phi1, phi2);
+            float dPhi = reco::deltaPhi(phi1, phi2);
```

Comment:
@VourMa 
remind me please the rationale of using `reco::deltaPhi(T phi1, T phi2)` vs `cms::alpakatools::deltaPhi(TAcc const& acc, T phi1, T phi2)`

Reply:
Closely following what was there before. That said, I am not sure why we ended up with `calculate_dPhi(phi1, phi2)` and not something `alpaka` related.

Reply:
uhm, but isn't the implementation different (the old was a single 2π shift vs new using somewhat pseudo-constexpr reducedRange), 
unless by "closely" you mean "it didn't have acc" before

Reply:
About following closely, it is the latter.
About your first point, my understanding is that using `reduceRange` is safer, as it accounts for multiple "wraparounds" of φ, but let me know if I didn't get it right.

Reply:
I'd aim to use the same code that defines dphi from x,y "directly" and after computing phi. My concern is that the computation path x,y -> phi  followed by a call to dPhi for a pair of items is different from computing it directly from a pair of two `x,y`s in the same parts of the code base.  ... also I'm still bothered by pseudo-constexpr in the reco case as was discussed with Matti.

Reply:
> I don't see an inconsistency, at least not in this part of the code, as we do not compute φ from x, y

that's not what I see: `quintuplets.phi()` defines the phi1,2 here, which in turn eventually comes from `hits.phis()[ihit] = phi(acc, ihit_x, ihit_y)`

Reply:
> but I understand that you would want us to apply `cms::alpakatools::deltaPhi(TAcc const& acc, T x1, T y1, T x2, T y2)` and not `cms::alpakatools::deltaPhi(TAcc const& acc, T phi1, T phi2)`?

no; if we have to cache phi, as in this case of T5s or MDs, they should be cached. I mean, eventually, that `cms::alpakatools::deltaPhi` should be preferred, especially if the inputs are likely also computed with `cms::alpakatools::` or accelerator-dependent code

Reply:
Ok, my bad, I thought it was pT5s we were looking at.

Do you want to change this? If so, do you want to do it in this PR?

Reply:
> no; if we have to cache phi, as in this case of T5s or MDs, they should be cached. I mean, eventually, that `cms::alpakatools::deltaPhi` should be preferred, especially if the inputs are likely also computed with `cms::alpakatools::` or accelerator-dependent code

Ok. Would you like to make a comment in the cms-sw PR to mention that we'd better use the alpaka version, and then I can follow up on that?

Reply:
> Do you want to change this? If so, do you want to do it in this PR?

it's better in this PR

Reply:
> Ok. Would you like to make a comment in the cms-sw PR to mention that we'd better use the alpaka version, and then I can follow up on that?

I stayed away from bringing this up in the cms-sw PR due to the review availability issues for heterogeneous, where weeks long delays are typical

Reply:
Ok, then I will force push the `reco::deltaPhi(T phi1, T phi2)` -> `cms::alpakatools::deltaPhi(TAcc const& acc, T phi1, T phi2)` change a bit later today.